match/lib/match/generator.rb

```diff
@@ -39,7 +39,13 @@ def self.generate_provisioning_profile(params: nil, prov_type: nil, certificate_
 
       prov_type = Match.profile_type_sym(params[:type])
 
-      profile_name = ["match", profile_type_name(prov_type), app_identifier].join(" ")
+      names = ["match", profile_type_name(prov_type), app_identifier]
+
+      if params[:platform] != 'ios' # For ios we do not include the platform for bc
```

Comment:
Yea currently match only does iOS profiles and it does not end them with the platform name. So i figured keep all iOS profiles un suffixed and make all tvOS (or mac in the future) have a suffix.